docs/code-reviews/process-guidance/README.md

```diff
@@ -22,6 +22,30 @@ If the team is finding that code reviews are taking a significant time to merge,
 1. Assess the time to merge across sprints to see if the process is improving.
 1. Ping required approvers directly as a reminder.
 
+## Recommendations for a better code review
+
+### Get a process for fixing found issues
+
+Before getting started with any commit, your team should define how you will measure the effectiveness of peer review and get some goals. Using a [SMART criteria](https://en.wikipedia.org/wiki/SMART_criteria) the team can get this objectives in the most accurate way and not something very general like **fix more bugs**, the more precise a goal can be the easier it will be to hit the target.
+
+Metrics can play a significant role in this part, you can include an **Inspection rate**(the speed with which review is performed), **Defect rate**(the number of bugs found within an hour of review) and **Defect density**(the number of bugs found per piece of code) as the most common ones to be considered.
+
+### Code reviews shouldn't include many lines of code
```

Comment:
This sounds like an author specific guidance, can we move it there?

Reply:
I took the section to the author guidance.